package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@onebeyond/agnostic-cms-harmonizer",
-  "version": "0.2.2",
+  "version": "0.2.3",
```

Comment:
This is automatically handled by the [release-and-publish action](https://github.com/onebeyond/agnostic-cms-harmonizer/blob/main/.github/workflows/release-and-publish.yml. No manual update needed.

Remember to revert changes in `package-lock.json` file too.